uprotocol.js

```diff
@@ -13,13 +13,11 @@
     Copyright 2018 John Maloney, Bernat Romagosa, and Jens Mönig
```

Comment:
If possible, I think this file should remain untouched. It was first written for Snap! and it follows its code style, which is Crockford's original JSLint.

Reply:
That's fair.